roles/fio-bench/templates/job.fio.rand

```diff
@@ -35,9 +35,9 @@ write_lat_log=fio
 write_hist_log=fio
 numjobs={{fio.numjobs}}
 per_job_logs=1
-log_avg_msec=60000
-log_hist_msec=60000
-startdelay={{ fio.runtime * 2 }}
+log_avg_msec={{fio.log_sample_rate}}
+log_hist_msec={{fio.log_sample_rate}}
+startdelay={{ fio.runtime + fio.ramp_time }}
```

Comment:
I don't think incrementing the startdelay is the right way to do this. Using `stonewall` here makes much more sense.

Reply:
stonewall would be good, but someone needs to verify that it is actually working the way we think it does if they haven't already. This can be done outside of ripsaw.  It's a little tricky do because the timestamps in the logs are relative to the start of the job, which is not same thing as relative to start of test.